src/layout.rs

```diff
@@ -80,8 +80,21 @@ impl Layout {
                 .background_suggestive_editors
                 .iter()
                 .skip_while(|(p, _)| p != &&path)
+                .skip_while(|(_, editor)| {
+                    editor.borrow().editor().buffer().owner() == BufferOwner::User
+                })
```

Comment:
Why?

Reply:
Say you perform a search and it finds your term in 40 files. When closing the window, it will go to the 39th found file, then the 38th, 37th, etc... Close seems like you opened 40 files.

Reply:
Oh, wait... hm. That should be `== BufferOwner::System`! Hm, my unit test is only opening two files. I see why it passed. I can fix.